src/ShopCurrencyBundle/Bridge/Chameleon/Migration/Script/update-1545308860.inc.php

```diff
@@ -0,0 +1,15 @@
+<h1>Build #1545308860</h1>
+<h2>Date: 2018-12-20</h2>
+<div class="changelog">
+    -#234 correct currency in order history
+</div>
+<?php
+
+
+$moduleManager = TCMSLogChange::getModuleManager('MTShopOrderHistory');
+$mapperConfig = $moduleManager->getMapperConfig();
+$mapperConfig->removeMapper('standard','chameleon_system_shop_currency.mapper.shop_currency_mapper');
+$mapperConfig->addMapper('standard','chameleon_system_shop_currency.mapper.shop_currency_mapper');
```

Comment:
What is the thought behind this change? Is it to move the mapper to the end of the list?

Reply:
Yes because the currency mapper loads the active currency if not set. But we need the currency from the order.